data/actionsmap/yunohost.yml

```diff
@@ -240,132 +250,82 @@ user:
+                    api: GET /users/permissions/<permission>
                     arguments:
-                        -a:
-                            full: --app
-                            help: Application to manage the permission
-                            nargs: "*"
-                            metavar: APP
-                        -p:
-                            full: --permission
-                            help: Name of permission (main by default)
-                            nargs: "*"
-                            metavar: PERMISSION
-                        -u:
-                            full: --username
-                            help: Username
-                            nargs: "*"
-                            metavar: USER
-                        -g:
-                            full: --group
-                            help: Group name
-                            nargs: "*"
-                            metavar: GROUP
+                        -s:
+                            full: --short
+                            help: List only the names of permissions
+                            action: store_true
+                        -f:
+                            full: --full
+                            help: Display all informations known about each permissions, including the full list of users corresponding to allowed groups.
+                            action: store_true
 
-                ### user_permission_add()
-                add:
-                    action_help: Grant access right to users and group
-                    api: POST /users/permissions/<app>
-                    arguments:
-                        app:
-                            help: Application to manage the permission
-                            nargs: "+"
-                        -p:
-                            full: --permission
-                            help: Name of permission (main by default)
-                            nargs: "*"
-                            metavar: PERMISSION
-                        -u:
-                            full: --username
-                            help: Username
-                            nargs: "*"
-                            metavar: USER
-                            extra:
-                                pattern: *pattern_username
-                        -g:
-                            full: --group
-                            help: Group name
-                            nargs: "*"
-                            metavar: GROUP
-                            extra:
-                                pattern: *pattern_username
 
-                ### user_permission_remove()
-                remove:
-                    action_help: Revoke access right to users and group
-                    api: PUT /users/permissions/<app>
+                ### user_permission_update()
+                update:
+                    action_help: Grant / remove permissions to groups or users
```

Comment:
```suggestion
                    action_help: Grant / remove group permissions, or users
```

Reply:
The idea was to keep "group" and "user" together since you can use both ... idk, or maybe let's rework the whole string with e.g. "Manage permissions accesses"

Reply:
I went half-and-half, accesses could mean a lot of other things.